Common/include/linear_algebra/CSysSolve.hpp

```diff
@@ -259,7 +259,7 @@ class CSysSolve {
    */
   unsigned long BCGSTAB_LinSolver(const VectorType & b, VectorType & x, ProductType & mat_vec,
                                   PrecondType & precond, ScalarType tol, unsigned long m,
-                                  ScalarType *residual, bool monitoring, CConfig *config);
+                                  ScalarType *residual, bool monitoring, CConfig *config, bool TapeActive);
```

Comment:
Please take this off the function signature, AD:: is also available inside the method